climakitae/data_export.py

```diff
@@ -457,3 +457,109 @@ def _rchop(s, suffix):
             ):
                 f.write(str(att_keys) + " : " + str(att_values))
                 f.write("\n")
+
+
+## TMY export functions
+def tmy_header(station_name, df):
+    """
```

Comment:
Can you rename this as `_tmy_header` since it's an internal (not user-facing) function?